Add more splash messages!

MegaTKC opened the pull request.

MegaTKC committed:
Add more splash messages!

torralbaa reviewed (commented):
Hi! Thanks for the contribution! But it has a few (easily fixable) errors:
 + You have to put a `,` before each new line (and the last of the old ones) except the last one.
 + Please use the same indentation level and style (tabs) of the other splash messages, and the same "**N**ormal **s**entence **c**apitalization" (this is purely stylistic though).
 + Please remove the invalid/expired Discord invite link. We already have a splash text for an invite to the server anyways.

Also, Google Translate says the Japanese greeting actually says "Japan hello!", but I don't know how much it should be trusted, so please confirm it's correct.

MegaTKC committed:
Update play.vala

MegaTKC commented:
> Hi! Thanks for the contribution! But it has a few (easily fixable) errors:
> 
> * You have to put a `,` before each new line (and the last of the old ones) except the last one.
> * Please use the same indentation level and style (tabs) of the other splash messages, and the same "**N**ormal **s**entence **c**apitalization" (this is purely stylistic though).
> * Please remove the invalid/expired Discord invite link. We already have a splash text for an invite to the server anyways.
> 
> Also, Google Translate says the Japanese greeting actually says "Japan hello!", but I don't know how much it should be trusted, so please confirm it's correct.

Hello, I have fixed the indentation, removed the Discord invite link and added those commas. For the Japanese greeting that says Japan hello! is correct. Minecraft also has this text that says Japan hello! and they use it for their game.

Alvarito050506 commented:
> Hello, I have fixed the indentation, removed the Discord invite link and added those commas.

Thanks!

> For the Japanese greeting that says Japan hello! is correct. Minecraft also has this text that says Japan hello! and they use it for their game.

Ohh, didn't know it was intentional... Merging this, thanks again.
